src/evaluate.jl

```diff
@@ -312,7 +312,10 @@ function evaluate_test(config::Configuration, pkg::Package; use_cache::Bool=true
 
         t0 = cpu_time()
         try
-            Pkg.add(; package_spec...)
+            # disable precompilation to avoid precompiling with the wrong options
+            withenv("JULIA_PKG_PRECOMPILE_AUTO" => false) do
```

Comment:
Worth checking that a bare false is parsed properly

Reply:
Yep this is good
```
pkg> activate --temp
  Activating new project at `/var/folders/_6/1yf6sj0950vcg4t91m9ltb5w0000gn/T/jl_Bqi25G`

shell> rm -rf ~/.julia/compiled/v1.8/CSV

julia> withenv("JULIA_PKG_PRECOMPILE_AUTO" => false) do
       Pkg.add("CSV")
       end
# no autoprecomp
```

Reply:
I hadn't realised withenv converted false to "false"
```
julia> withenv("JULIA_PKG_PRECOMPILE_AUTO" => false) do
       ENV["JULIA_PKG_PRECOMPILE_AUTO"]
       end
"false"
```

Reply:
Thanks for checking!